src/style.css

```diff
@@ -1 +1,20 @@
 /* styles go here */
+.form-field {
+  display: block;
+}
+
+input {
+  margin-top: 10px;
+}
+
+.inp {
+  margin-top: 0;
+}
+
+fieldset {
+  margin-bottom: 20px;
+}
```

Comment:
you can use `:not(:last-child)` to style all fieldsets/labels besides last one.
Example:
```
.filedset:not(:last-child) {
  margin-bottom: 20px
}

.filed:not(:last-child) {
  margin-bottom: 10px
}
```